chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomsStateDifferTest.kt

```diff
@@ -0,0 +1,64 @@
+package com.pusher.chatkit.rooms.state
+
+import assertk.assertThat
+import assertk.assertions.containsExactly
+import com.pusher.chatkit.state.LeftRoom
+import com.pusher.chatkit.state.ReconnectJoinedRoom
+import com.pusher.chatkit.state.RoomUpdated
+import com.pusher.chatkit.state.State
+import io.mockk.every
+import io.mockk.mockk
+import org.reduxkotlin.GetState
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class JoinedRoomsStateDifferTest : Spek({
+
+    describe("given one room") {
+        val state = mockk<GetState<State>>(relaxed = true)
+        every { state().joinedRoomsState } returns JoinedRoomsState(
+            rooms = mapOf(roomOneId to roomOne),
+            unreadCounts = mapOf(roomOneId to 1))
```

Comment:
```suggestion
            unreadCounts = mapOf(roomOneId to 1)
        )
```
minor formatting (consistency, readability)